docs/core_tenets.md

```diff
@@ -0,0 +1,18 @@
+## Jujutsu's Core Tenets
+
+Jujutsu's core tenets are:
+
+ * User-friendliness: Making the  working copy a commit is simpler. This is
+ how the project started.
+ * The "repository", so the commit graph is the source of truth. The working
+ copy is just one way of editing commits.
+ * All operations must be able to scale to Google-scale repos (lots of commits
+ , lots of files): Laziness is important, must avoid accessing data
```

Comment:
nit: don't break the line just before `,`

Reply:
done